constants/tasks.ts

```diff
@@ -44,6 +44,10 @@ const COMPLETED_TASK_STATUS = {
 };
 const TASK_SIZE = 5;
 
+const tasksUsersStatus = {
+  MISSED_UPDATES: "missed-updates",
```

Comment:
Any reason to have added this as a part of an object rather than just a primitive string? Because I see that you are accessing `tasksUsersStatus.MISSED_UPDATES`. Are there plans to  have more tasksUsersStatus in the future?

Reply:
Yes we can have.